### PR TITLE
Add support to get the last modified time from the filesys

### PR DESCRIPTION
**Pull request by bheni**

### CODE REVIEW

File: go/libraries/utils/filesys/inmemfs.go [hunk 1]
```diff
@@ -108,8 +120,10 @@ func NewInMemFS(dirs []string, files map[string][]byte, cwd string) *InMemFS {
 				panic("Initializing InMemFS with invalid data.")
 			}
 
-			newFile := &memFile{path, val, targetDir}
+			now := time.Now()
```
Comment (zachmu) [inline go/libraries/utils/filesys/inmemfs.go]: If we're going to the trouble of having an in-memory file system, we probably want to have it use a fake clock for this kind of thing. Maybe in a second pass? That would make it easier to write repeatable tests for mtime.

File: go/libraries/utils/filesys/inmemfs.go [hunk 2]
```diff
@@ -460,17 +476,20 @@ func (fs *InMemFS) MoveFile(srcPath, destPath string) error {
 			return err
 		}
 
-		destObj := &memFile{destPath, obj.(*memFile).data, destParentDir}
+		now := time.Now()
+		destObj := &memFile{destPath, obj.(*memFile).data, destParentDir, time.Now()}
```
Comment (zachmu) [inline go/libraries/utils/filesys/inmemfs.go]: Should use now instead of a new call to time.Now()

Comment (bheni) [reply go/libraries/utils/filesys/inmemfs.go]: fixed

File: go/libraries/utils/filesys/inmemfs.go (hunk 1 above)
Comment (bheni) [reply go/libraries/utils/filesys/inmemfs.go]: Put in something quick and dirty.

**Pull request merged**